pkgs/os-specific/linux/rtl8812au/default.nix

```
@@ -42,8 +42,8 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Driver for Realtek 802.11ac, rtl8812au, provides the 8812au mod";
-    homepage = "https://github.com/gordboy/rtl8812au-5.6.4.2";
-    license = licenses.gpl2;
+    homepage = "https://github.com/gordboy/rtl8812au-5.9.3.2";
+    license = licenses.gpl2Only;
     platforms = platforms.linux;
     maintainers = with maintainers; [ danielfullmer ];
```

```suggestion
    maintainers = with maintainers; [ danielfullmer ];
    broken = kernel.kernelOlder "4.10" || kernel.isHardened;
```

to remove unsupported kernels.